std_msgs/Bool command
---
RobotState state
